base/gmp.jl

```diff
@@ -74,8 +74,8 @@ Create an arbitrary precision integer. `x` may be an `Int` (or anything that can
 converted to an `Int`). The usual mathematical operators are defined for this type, and
 results are promoted to a [`BigInt`](@ref).
 
-Instances can be constructed from strings via [`parse`](@ref), or using the `big`
-string literal.
+Instances can be constructed from strings via [`parse`](@ref) or at macro expansion time
+via the `big` string literal.
```

Comment:
IMO "at macro expansion time" isn't very useful. It doesn't add anything for those who are already aware of the implications of string literals being macros, and those who are *not* aware of macro pitfalls will likely not even realize this subtle warning in the text. Maybe you could use an actual warning element (displayed with a red background), like in the related #50800?

Also, you should probably use cross-references, like
```
[`@big_str`](@ref)
```

instead of just
```
`big`
```